importInvoice.js

```diff
@@ -0,0 +1,157 @@
+function importInvoice({ ramda: R }) {
+  const re = /([0-9]+:[0-9]+[ -]+[0-9]+:[0-9]+)/g;
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  function tag(tagName) {
+    return content => `<${tagName}>` + content + `</${tagName}>`;
+  }
+  function zeroPrefix(num) {
+    if (num <= 9) {
+      return `0${num}`;
+    }
+    return `${num}`;
+  }
+  function formatDate(date) {
+    return [date.getDate(), months[date.getMonth()], date.getFullYear()].join(' ');
+  }
+  function sum(arr) {
+    return arr.reduce((acc, num) => acc + num, 0);
+  }
+  function replaceLine(line) {
+    const segmented = R.flatten(
+      R.splitEvery(2, line.split(re)).map(([text, match]) => {
+        if (!match) {
+          return text;
+        }
+        const [start, end] = match.replace(' ', '').split('-');
+
+        const [startHours, endHours] = [start, end].map((time) => {
+          const [hr, min] = time.split(':').map((v) => parseInt(v, 10));
+          return hr + min / 60;
+        });
+        const duration = endHours >= startHours ? endHours - startHours : endHours + 24 - startHours;
+        return [text, { timeRange: match, hours: duration }];
+      })
+    );
+    const dateColumn = segmented[0];
+    const timeRange = R.dropLast(1, R.drop(1, segmented))
+      .map((item) => (item.timeRange || item))
+      .join('');
+    const description = R.last(segmented);
+    const totalHoursPerRow = sum(segmented.map((item) => item.hours || 0));
+    return [dateColumn, timeRange, description, totalHoursPerRow];
+  }
+  function renderInvoice(invoiceDate, text, adjustments = [], invoicePrefix = 'OX', client = { company: 'Orexplore Technologies', email: 'accounts@orexplore.com', phone: '(08) 9469 2900' }) {
+      const hourlyWage = 80; // Example hourly wage baed on screenshot, you can adjust this value
+      const taxPercent = 10; // Percent to be added at the end
```

Comment:
`importInvoice.js` is a shared library that should handle both rendering tax and non-tax invoices, currently it looks like it's hardcoded to do tax. So you will need to pass `taxPercent` as argument for `renderInvoice`. Probably pass `hourlyWage` will you are at it. Get rid of all the orexplore company details, use dummy ones if you nede to.